### PR TITLE
Event creation flow

### PR DESCRIPTION
**Pull request by yashjain1394**

### CODE REVIEW

File: src/e2e/features/createEvent.feature
Comment (gbajaj91) [inline src/e2e/features/createEvent.feature:1]: lets divide it into two scenraios, first being the dashboard verification, and next for creation

File: src/e2e/config/test-data/constants.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const expectedTableHeaders = [
```
Comment (gbajaj91) [inline src/e2e/config/test-data/constants.js]: lets add this in the definition file only

Comment (yashjain1394) [reply src/e2e/config/test-data/constants.js]: @semwalseema As per discussion, done this change

File: src/e2e/features/createEvent.feature
Comment (yashjain1394) [reply src/e2e/features/createEvent.feature:1]: @semwalseema As per discussion, done this change

**Review by semwalseema — APPROVED** [review]

**Pull request merged**